TurkishFilter/sections/specific.txt

```diff
@@ -110,6 +110,7 @@ webteizle.vip###sol > .ui > a[href][target="_blank"][rel="nofollow"]
 muratlihizmetgazetesi.com##img[alt^="reklam"]
 muratlihizmetgazetesi.com##.beyaz > div > p[style="text-align:center"] > img
 ceptenmuzikindir.me##.contentbox > center > div > a[rel="nofollow"]
+kpssrehber.com##a[target="_blank"][href^="https://goo.gl/"]
```

Comment:
Why not a `kpssrehber.com##a[href^="https://goo.gl/"]`?

Reply:
I like [target="_blank"] selectors for some reason